corehq/apps/es/es_query.py

```diff
@@ -459,7 +471,7 @@ def get_ids(self):
 
     def scroll_ids(self):
         """Returns a generator of all matching ids"""
-        return self.exclude_source().size(5000).scroll()
```

Comment:
Doesn't excluding source change the performance characteristics of this significantly, versus a normal scroll query?  I would've expected to see different limits between the two types, though I haven't done any research to back that up.